src/newrelic_airflow_plugin/newrelic_plugin.py

```diff
@@ -59,7 +59,8 @@ def harvester(cls):
                 return harvester
 
             insert_key = os.environ["NEW_RELIC_INSERT_KEY"]
-            client = MetricClient(insert_key)
+            host = os.environ.get("NEW_RELIC_HOST", "metric-api.newrelic.com")
```

Comment:
```suggestion
            host = os.environ.get("NEW_RELIC_HOST", None)
```

Leave this as None so that the default value in the MetricsClient is still used instead of duplicating that here.